django/contrib/admin/static/admin/js/vendor/xregexp/LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2007-present Steven Levithan <http://xregexp.com/>
```

Comment:
This is the LICENCE that should be used directly without any modifications on our side, please revert.